src/components/AboutPage.js

```diff
@@ -59,74 +62,87 @@ const AboutPage = () => (
           <li>...</li>
           <li>...</li> */}
         </ul>
+        <p>
+          Contact us if you would like to get involved.
+        </p>
       </Col>
     </Row>
 
     <hr />
     
     <Row>
-      <Col className="col"></Col>
-      <Col className="col-8">
+      <Col>
         <h2>Request a Project</h2>
         <p>
           Are you a black business owner, activist, protest organizer, or 
           community advocate looking for help building a website or online 
           platform, creating social media content, data visualizations, or other
            projects?
         </p>
-        <Col>
-          <LinkContainer to="/project-request">
-            <Button className="button" variant="warning" size="med">
-              Request a Project
-            </Button>
-          </LinkContainer>
-        </Col>
+        <LinkContainer to="/project-request">
+          <Button className="button" variant="warning" size="med">
+            Request a Project
+          </Button>
+        </LinkContainer>
       </Col>
     </Row>
 
     <hr />
 
     <Row>
-      <Col className="col"></Col>
-      <Col className="col-10">
+      <Col>
         <h2>Work on a Project</h2>
         <p>
           Are you a designer, programmer, engineer, data scientist, writer, web 
           developer, or marketing specialist looking to use your skills to do 
           pro bono work to support Black lives?
         </p>
-        <Col>
-          <LinkContainer to="/projects">
-            <Button className="button" variant="warning" size="med">
-              Work on a Project
-            </Button>
-          </LinkContainer>
-        </Col>
+        <LinkContainer to="/projects">
+          <Button className="button" variant="warning" size="med">
+            Work on a Project
+          </Button>
+        </LinkContainer>
       </Col>
     </Row>
 
     <hr />
 
     <Row>
-      <Col className="col"></Col>
-      <Col className="col-8">
+      <Col>
         <h2>Contact Us</h2>
-        {/* TODO replace with icons */}
+        <p>
+          Feel free to reach out to us if you are interested in getting involved
+           or have questions! You can also check out our HackBeanpot social 
+           media pages for more information about what we do as an organization.
+        </p>
         <a href="mailto:buildforblacklives@hackbeanpot.com">
           buildforblacklives@hackbeanpot.com
-        </a> <br />
-        <a href="https://www.facebook.com/hackbeanpot/">
-          Facebook
-        </a> <br />
-        <a href="https://www.instagram.com/hackbeanpot/">
-          Instagram
-        </a> <br />
-        <a href="https://www.linkedin.com/company/hackbeanpot-inc">
-          LinkedIn
-        </a> <br />
-        <a href="https://twitter.com/hackbeanpot">
-          Twitter
         </a>
+        <br />
+        <a href="https://hackbeanpot.com/"
+        target="_blank" rel="noopener noreferrer">
+          hackbeanpot.com
```

Comment:
Could we add text around these maybe? (we can try to fix how links show up in a later style runthrough)
Email us at <email>
Visit us at <website>

Reply:
same line or preceding line?